types/types.go

```diff
@@ -36,12 +36,24 @@ type NetConf struct {
 	CNIDir  string `json:"cniDir"`
 	BinDir  string `json:"binDir"`
 	// RawDelegates is private to the NetConf class; use Delegates instead
-	RawDelegates []map[string]interface{} `json:"delegates"`
-	Delegates    []*DelegateNetConf       `json:"-"`
-	NetStatus    []*NetworkStatus         `json:"-"`
-	Kubeconfig   string                   `json:"kubeconfig"`
-	LogFile      string                   `json:"logFile"`
-	LogLevel     string                   `json:"logLevel"`
+	RawDelegates  []map[string]interface{} `json:"delegates"`
+	Delegates     []*DelegateNetConf       `json:"-"`
+	NetStatus     []*NetworkStatus         `json:"-"`
+	Kubeconfig    string                   `json:"kubeconfig"`
+	LogFile       string                   `json:"logFile"`
+	LogLevel      string                   `json:"logLevel"`
+	RuntimeConfig *RuntimeConfig           `json:"runtimeConfig,omitempty"`
+}
+
+type RuntimeConfig struct {
+	PortMaps []PortMapEntry `json:"portMappings,omitempty"`
+}
+
+type PortMapEntry struct {
```

Comment:
Do we want to add testing to https://github.com/intel/multus-cni/blob/master/types/conf_test.go to verify that the PortMapEntry is properly populated?

Reply:
I just added a check to ensure the PortMaps struct is not empty when a runtimeConfig is given